tox.ini

```diff
@@ -17,6 +17,6 @@ deps =
     -rrequirements.txt
     pyinstaller
 commands =
-    pyinstaller --clean -y -F -n shub --additional-hooks-dir=./freeze/hooks --icon=./freeze/spider-down.ico ./freeze/shubrunner.py
+    pyinstaller --clean -y -F -n shub --additional-hooks-dir=./freeze/hooks --runtime-hook=./freeze/hooks/runtime-hooks.py --icon=./freeze/spider-down.ico ./freeze/shubrunner.py
     {toxinidir}/dist/shub version
```

Comment:
is it too crazy if we include all test dependencies in the bundle and add a command like `shub run-tests` ?